src/graphql-server/index.js

```diff
@@ -2,8 +2,6 @@ import { makeExecutableSchema } from 'graphql-tools'
 import typeDefs from './schema.graphql'
 import resolvers from './resolvers'
 
-console.log('1 - Removing this log statement breaks the app. Investigate why...')
```

Comment:
Hey @robbiecarlton, did you discover why? :D

Reply:
It's something to do with webpack/graphql/caching. The issue is def not resolved, it's just I got it to work without that line. Would be great to pair with someone on the underlying webpack issue.

Reply:
Perhaps it has something to do with webpack (or sth else) checking if the module has sideeffects - console is global, so it would probably trigger such check 🤔